### PR TITLE
radeontop: 2018-06-03 -> 1.3

### PR DESCRIPTION
**Pull request by lovesegfault**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/os-specific/linux/radeontop/default.nix [hunk 1]
```diff
@@ -3,11 +3,11 @@
 
 stdenv.mkDerivation rec {
   pname = "radeontop";
-  version = "2019-06-03";
+  version = "1.3";
```
Comment (fabaff) [inline pkgs/os-specific/linux/radeontop/default.nix:6]: I'm not expert on how Nix is handling the sorting. At first glance it looks like that users will likely still end-up with `20xx` as it comes after `1.x`.

If I'm wrong then I will take it take and blame the Fedora package maintainer in me.

**Review by fabaff — COMMENTED** [review]
Another small thing: Commit message should be 

```text
radeontop: 2019-06-03 -> 1.3
```
See https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes

**Comment by lovesegfault** [comment]
> Another small thing: Commit message should be
> 
> ```
> radeontop: 2019-06-03 -> 1.3
> ```
> 
> https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes
> See

It was a typo. Fixed.

File: pkgs/os-specific/linux/radeontop/default.nix (hunk 1 above)
Comment (lovesegfault) [reply pkgs/os-specific/linux/radeontop/default.nix:6]: That's incorrect, from Nix's POV packages with different versions are just different packages.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 107868` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>radeontop</li>
  </ul>
</details>

File: pkgs/os-specific/linux/radeontop/default.nix (hunk 1 above)
Comment (lovesegfault) [reply pkgs/os-specific/linux/radeontop/default.nix:6]: To clarify, only one version of a package exists at a time in nixpkgs. If two versions need to exist, they're made into different packages and used accordingly (like `gcc`, `gcc7`, `gcc8`, etc)

Comment (fabaff) [reply pkgs/os-specific/linux/radeontop/default.nix:6]: Thanks for the clarification.

**Review by fabaff — APPROVED** [review]

File: pkgs/os-specific/linux/radeontop/default.nix (hunk 1 above)
Comment (rycee) [reply pkgs/os-specific/linux/radeontop/default.nix:6]: The sorting is important for people who use `nix-env -u`, that's the reason why I kept the date format even after radeontop started with named versions.

**Pull request merged**